stock_putaway_by_route/models/stock_move.py

```diff
@@ -18,7 +18,7 @@ def _generate_serial_move_line_commands(self, lot_names, origin_move_line=None):
         )
 
     def _get_putaway_routes(self):
-        routes = self.env["stock.location"]
+        routes = self.env["stock.location.route"]
```

Comment:
Even if 
```suggestion
        routes = self.env["stock.location.route"].browse()
```
is better